app/lib/validation-helpers.js

```diff
@@ -1,7 +1,11 @@
+const Joi = require('joi')
 const { isFuture, isWithinInterval, sub, addMonths, startOfDay, differenceInYears } = require('date-fns')
 const { parseDate } = require('./date-helpers')
+const { getPersonAndDogs } = require('../api/ddi-index-api/person')
 const validNewMicrochip = /^\d+$/
 
+const invalidBreedForCountryMessage = 'The address for an XL Bully dog must be in England or Wales'
```

Comment:
would this be better in the consts directory?

Reply:
As it's a validation message, I think it better sits where it is (in a validation helper) - unless we start pulling all error messages out into a consts file